napari/_qt/qt_main_window.py

```diff
@@ -130,6 +135,7 @@ def __init__(self, viewer, *, show: bool = True):
         self.qt_viewer = QtViewer(viewer)
 
         self._qt_window = QMainWindow()
+        self._qt_window.setWindowIcon(QIcon(logopath))
```

Comment:
Do we still need this?

Reply:
If I good remember windows show a window icon on the left side of the title bar.